ci/Dockerfile.fuel-node

```diff
@@ -31,7 +31,7 @@ COPY --from=planner /build/recipe.json recipe.json
 ENV SQLX_OFFLINE=true
```

Comment:
- Everywhere in the code, the test node for the `hello-world` example is referenced as `hello-world-test-node`. 
- Could we update this filename to reflect that? It seems to still have the old/generic `fuel-node` name.